source/scenario/mission2task6.vwf.yaml

```diff
@@ -107,7 +107,6 @@ children:
             - "Use a loop to move along the slope of the tracks two more times."
           - setThirdPersonStartPose:
             - [ 16.316, 65, -15 ]
-          - startTimer:
```

Comment:
Yeah... I'm pretty sure that that's actually what we want, though.  If it takes 5 tries to get through a scenario, the timer should count the total time.

With that said... we should keep an eye out for issues that come out of this.